packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -316,7 +316,7 @@ const Messages: FunctionComponent<Props> = ({
       )
       if (thread) {
         openThreadDetails(thread)
-      } else {
+      } else if (tmpActiveThread === undefined && thread === undefined) {
```

Comment:
(giving it here, as it is easier to have a thread)

Not sure if this is a scope of this task, but seems to me that when you write a message and then select a contact, the message disappears. Is it expected?

https://user-images.githubusercontent.com/41268731/171599734-0125f1ce-0e6f-4355-b450-c5e1d941385f.mov

Reply:
> Is it expected?

Looks like space for improvement. However,  it isn't the scope of these bugs. Let's cover it as part of the next issue 😉